src/Services/Utils/UtilsValidator.js

```diff
@@ -27,11 +28,31 @@ Validator.prototype.currencies = function () {
   return this;
 };
 
+Validator.prototype.datatype = function () {
+  if (Object.prototype.toString.call(this.params.dataType) !== '[object String]') {
+    throw new UtilsValidationError.DataTypeMissing(this.params);
+  }
+
+  if (this.params.dataType.length <= 0) {
+    throw new UtilsValidationError.DataTypeMissing(this.params);
+  }
+
+  if (referencedatatypes.indexOf(this.params.dataType) === -1) {
```

Comment:
let's use this fancy `if (!array.includes(item)` syntax, what do you think?

Reply:
Made it that way..